jaeger/charts/linkerd-jaeger/values.yaml

```diff
@@ -1,6 +1,9 @@
 # -- Namespace of the Linkerd core control-plane install
 linkerdNamespace: linkerd
 
+# -- Docker imagePullPolicy for all jaeger components
+defaultImagePullPolicy: IfNotPresent
```

Comment:
Can you rename this to `imagePullPolicy` to match what we have in the `linkerd-control-plane` chart? (Also for the multicluster chart)

Reply:
Hi @alpeb, It's done.